### PR TITLE
DOM project (re-uploaded)

### PR DESCRIPTION
**Pull request by kovalov**
### DOM

[Demo](https://kovalov.github.io/kottans-dom-project/) |
[Code base](https://github.com/kovalov/kottans-dom-project)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/kovalov/dom/scripts/element.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export function createElement({ name, description, src }) {
+  const HTML = `
+	 <div>
+			 <h1 class="main__title">${name}</h1>
+			 <img class="main__image" src="./assets/images/${src}" alt="Photo of ${name}.">
+			 <p class="main__description">${description}</p>
+	 </div>`;
+  const template = document.createElement('template');
+  template.innerHTML = HTML;
+  return template.content.firstElementChild;
```
Comment (Roophee) [inline submissions/kovalov/dom/scripts/element.js:10]: Why are you using Template Element? It is makes sense if `<template>` existing into your html and you get something from there to push into html.  But you don't do this. It is better create `<div>` or `fragment` and return it.
It looks realy strenge.

**Review by Roophee — COMMENTED** [review]
Good job!!! I hope my comments help to you make your PR better.

File: submissions/kovalov/dom/scripts/element.js (hunk 1 above)
Comment (kovalov) [reply submissions/kovalov/dom/scripts/element.js:10]: Have used template to prevent creation of a few elements.

Comment (Roophee) [reply submissions/kovalov/dom/scripts/element.js:10]: You can create `DIV` and pass your template string with data item values as `innerHTML` and return  this `DIV` from your element creator.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
You submission contains unrelated code (see [Files changed](./files)) from a different task.

You probably missed to create a dedicated branch for either or both of your tasks.

Please fix this issue.

**Pull request closed without merging**